minihack/agent/common/envs/wrapper.py

```diff
@@ -122,20 +123,20 @@ def __init__(self, env):
         self._actions = self.env.actions
 
     def step(self, action):
-        next_state, reward, done, info = self.env.step(action)
+        next_state, reward, terminated, truncated, info = self.env.step(action)
         next_state["prev_reward"] = np.array([reward], dtype=np.float32)
         next_state["prev_action"] = np.array([action], dtype=np.uint8)
 
         self.last_observation = next_state
 
-        return next_state, reward, done, info
+        return next_state, reward, terminated, truncated, info
 
-    def reset(self, wizkit_items=None):
-        obs = self.env.reset(wizkit_items=wizkit_items)
+    def reset(self, seed=None, options={}):
```

Comment:
`options={}` is actually dangerous here and everywhere else it repeats.

```suggestion
    def reset(self, seed=None, options=None):
```